OpenRA.Mods.Common/WorldExtensions.cs

```diff
@@ -71,6 +71,7 @@ public static IEnumerable<Actor> FindBlockingActorsOnLine(this World world, WPos
 
 		/// <summary>
 		/// Finds all the actors of which their health radius might be intersected by a specified circle.
+		/// Note: Results may include false positives.
```

Comment:
This alone doesn't seem very helpful. I assume this is because of `world.ActorMap.LargestActorRadius`? Can we add a bit of explanation here?

Reply:
This was already implied by the "might" in the sentence above, FWIW.

Reply:
Btw that "might" is making us do lots of unnecessary checks.

Reply:
It also saves a significant amount of unnecessary calculations in some important hot paths.

Reply:
My point is we should narrow this down further. I will remove the comment if it's such a problem.

Reply:
IMO removing the comment change is the simplest "solution" here.

Reply:
Removed.